cmake/caches/Windows-aarch64.cmake

```diff
@@ -26,15 +26,27 @@ set(LLVM_DEFAULT_TARGET_TRIPLE aarch64-unknown-windows-msvc CACHE STRING "")
 set(LLVM_APPEND_VC_REV NO CACHE BOOL "")
 set(LLVM_ENABLE_PER_TARGET_RUNTIME_DIR YES CACHE BOOL "")
 set(LLVM_ENABLE_PYTHON YES CACHE BOOL "")
-set(LLVM_RUNTIME_TARGETS
-      aarch64-unknown-windows-msvc
-    CACHE STRING "")
+
+set(DEFAULT_RUNTIME_TARGETS
+      aarch64-unknown-windows-msvc)
+# Build the android runtimes if NDK path is provided.
+if(NOT "$ENV{NDKPATH}" STREQUAL "")
+  list(APPEND DEFAULT_RUNTIME_TARGETS
+       aarch64-unknown-linux-android)
```

Comment:
We should include `x86_64-unknown-linux-android` I think to match the x64 builds.